hack/update-payload-crds.sh

```diff
@@ -21,7 +21,11 @@ crd_globs="\
     operator/v1/zz_generated.crd-manifests/0000_25_kube-controller-manager_01_kubecontrollermanagers*.crd.yaml
     config/v1/zz_generated.crd-manifests/0000_10_openshift-controller-manager_01_builds*.crd.yaml
     operator/v1/zz_generated.crd-manifests/0000_50_openshift-controller-manager_02_openshiftcontrollermanagers*.crd.yaml
-    machineconfiguration/v1/zz_generated.crd-manifests/*.crd.yaml
+    machineconfiguration/v1/zz_generated.crd-manifests/0000_80_machine-config_01_containerruntimeconfigs*.crd.yaml
+    machineconfiguration/v1/zz_generated.crd-manifests/0000_80_machine-config_01_controllerconfigs*.crd.yaml
+    machineconfiguration/v1/zz_generated.crd-manifests/0000_80_machine-config_01_kubeletconfigs*.crd.yaml
+    machineconfiguration/v1/zz_generated.crd-manifests/0000_80_machine-config_01_machineconfigpools*.crd.yaml
+    machineconfiguration/v1/zz_generated.crd-manifests/0000_80_machine-config_01_machineconfigs*.crd.yaml
```

Comment:
They will be once we switch the MCO code to use the new CRDs. Right now, we're just generating the new API here, and adapting the MCO code to use it with the new vendor. Then we will open a PR here to graduate the FG + have new new v1 CRDs generate, and test with both PRs.

The concern is that if I do generate with the new v1 CRDs off the bat, all the techpreview jobs would fail since the MCO is still using the old API/CRD. I could be mis-understanding that though! In that case, we can switch directly over (I made https://github.com/openshift/api/pull/2100/files to do that)

Reply:
Ahh right, yes, good point! In that case you're right, you'll need to do this in a later step. Can you add a TODO or something in here so we remember that this is why we are doing this please